supportedBackends/google/pipelines/common/src/test/resources/docker-image-cache-manifest.json

```diff
@@ -1,4 +1,10 @@
 {
-  "project1/dockerImage1": "projects/some-google-project/global/images/dockerCacheDiskForDockerImage1",
-  "project2/dockerImage2": "projects/another-google-project/global/images/dockerCacheDiskForDockerImage2"
+  "project1/dockerImage1": {
+    "dockerImageDigest": "sha256:c73e11b1a7854f31ff12c607738bef7b0560a880dc9d0445dde084acc0e9da09",
+    "diskImageName": "projects/some-google-project/global/images/dockerCacheDiskForDockerImage1"
+  },
+  "project2/dockerImage2": {
+    "dockerImageDigest": "sha256:7eb386481d87e41ebddceb948f25379bb339784df5247a3dfbdea2ac101b10c0",
+    "diskImageName": "projects/another-google-project/global/images/dockerCacheDiskForDockerImage2"
+  }
```

Comment:
Inspired by these changes, perhaps there should be a `version` field in this manifest format as well. I do like `version 2` for this as it is a breaking change from the implicit `version 1`.